src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionAzureKeyVaultProvider.java

```diff
@@ -56,7 +56,9 @@
     private final String azureTrustedEndpoints[] = {"vault.azure.net", // default
             "vault.azure.cn", // Azure China
             "vault.usgovcloudapi.net", // US Government
-            "vault.microsoftazure.de" // Azure Germany
+            "vault.microsoftazure.de", // Azure Germany
+            "vault.cloudapi.eaglex.ic.gov",
+            "vault.cloudapi.microsoft.scloud"
```

Comment:
Should add the trailing comma for cleaner diffs down the road when new endpoints are added.

Reply:
Thanks for the suggestion @sehrope , updated the PR.